test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +61,8 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertIsAnyNull(Object... objects) { assertTrue(Utils.isAnyNull(objects)); }
```

Comment:
You can consider just using `assertTrue.(Utils.isAnyNull(...))` in the method `isAnyNull` on Line 14 instead of abstracting this out as a method. (... contains what you want to test.)

The reason why existing tests do that (`assertAreUnique` and `assertNotUnique`) is to avoid repeating usage of `Arrays.asList`.

Same comment for the method `assertNotNull`.